static/css/impala/buttons.less

```diff
@@ -298,9 +298,44 @@ button[disabled],
     color: #919497;
     .box-shadow(0 3px rgba(0, 0, 0, 0.05), 0 -4px rgba(0, 0, 0, 0.05) inset);
     text-shadow: 0 1px 0 rgba(255,255,255,.5);
+    white-space: normal;
```

Comment:
I'm not sure, but it seems this file isn't used. At least not on the collections page. This CSS on AMO is a mess :/